cmd/goal/commands.go

```diff
@@ -492,36 +492,50 @@ func ensurePassword() []byte {
 }
 
 func reportInfoln(args ...interface{}) {
-	fmt.Println(args...)
-	// log.Infoln(args...)
+	for _, line := range strings.Split(fmt.Sprint(args...), "\n") {
+		printable, line := unicodePrintable(line)
+		if !printable {
+			fmt.Println(infoNonPrintableCharacters)
+		}
+		fmt.Println(line)
+	}
 }
 
 func reportInfof(format string, args ...interface{}) {
-	fmt.Printf(format+"\n", args...)
-	// log.Infof(format, args...)
+	reportInfoln(fmt.Sprintf(format, args...))
 }
 
 func reportWarnln(args ...interface{}) {
 	fmt.Print("Warning: ")
-	fmt.Println(args...)
-	// log.Warnln(args...)
+
```

Comment:
Why not just invoke `reportInfoln` here?

Reply:
I didn't want to mixup the info level handlers and the warning level handlers.